homeassistant/components/wyoming/stt.py

```diff
@@ -83,6 +91,11 @@ def supported_channels(self) -> list[stt.AudioChannels]:
         """Return a list of supported channels."""
         return [stt.AudioChannels.CHANNEL_MONO]
 
+    @property
```

Comment:
Should we still return `supported_languages` or have STT derive that from the models property when it's given?

Reply:
We may consider deprecating the `supported_languages` property altogether in favor of `supported_models`. The frontend should filter `supported_models` and only display the ones compatible with the pipeline language. Bundling all of the languages of all the models into one big set isn't useful for that purpose.